tests/Main.hs

```diff
@@ -0,0 +1,13 @@
+module Main
+  ( main
+  )
+where
+
+import Prelude
+
+import Freckle.App.Test.Hspec.Runner (runParConfig)
```

Comment:
Can't use this either, unfortunately. Just a standard `hspec-discover` in `Main.hs` should suffice for a `-library` template.